modopt/opt/algorithms.py

```diff
@@ -64,11 +64,12 @@ class SetUp(Observable):
 
     """
 
-    def __init__(self, metric_call_period=5, metrics={}, linear=None,
-                 verbose=False):
+    def __init__(self, metric_call_period=5, metrics={}, verbose=False,
+                 progress=True, **kwargs):
```

Comment:
Maybe we should mark then as `dummy_kwargs` to not have pep8 pick up on it, or be misled by thinking they might be used somewhere

Reply:
This should not be an issue for PEP8 as it is pretty standard to support `*args` and `**kwargs` without using them. 

Do you think labelling this a `dummy_kwargs` would help developers?

Reply:
well running my pylint on the file raises the fact that `kwargs` is unused ([see also](https://stackoverflow.com/questions/10025680/how-can-i-denote-unused-function-arguments)).

I think it's better to have `dummy_kwargs` in my experience, but even better to avoid that kind of situations. I understand it's a bit of a rush so I would go with `dummy_kwargs`.